apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -265,80 +254,85 @@ const DanceAiModal: React.FunctionComponent<DanceAiModalProps> = ({
       [FieldKey.BACKGROUND_EFFECT]: backgroundLabels,
       [FieldKey.BACKGROUND_PALETTE]: paletteLabels,
     };
-  };
-
-  const getAllItems = () => {
-    return inputLibraryJson.items.map(item => {
-      return {
-        ...item,
-        available: !inputs.includes(item.id),
-      };
-    });
-  };
-
-  const getItem = (id: string) =>
-    inputLibraryJson.items.find(item => item.id === id);
```

Comment:
This function was moved to `utils/getItem` (which should be named something more specific) and optimized.

Notes are in that function.